configs/get_dependencies.go

```diff
@@ -0,0 +1,17 @@
+package configs
+
+/*
+GetDependencies :
+This function is responsible for giving the dependencies needed for 1Click setup
+
+params :-
+None
+
+returns :-
+a. []string
+List of dependencies needed for 1Click setup
+*/
+func GetDependencies() []string {
+	//TODO: Get dependencies from config file
+	return Dependencies
```

Comment:
variable name convention to be followed --> lowercase

Reply:
This variable is hardcoded in messages.go for now. Will move this stuff to a config file in a way we need to discuss. For now, it must be uppercased because it needs to be exported